ops/demo/api.tf

```diff
@@ -12,17 +12,19 @@ module "simple_report_api" {
   https_only       = true
 
   app_settings = {
-    SPRING_PROFILES_ACTIVE                         = "azure-demo,no-security,no-okta-mgmt"
-    SPRING_LIQUIBASE_ENABLED                       = "true"
-    SPRING_JPA_PROPERTIES_HIBERNATE_DEFAULT_SCHEMA = "public"
-    SPRING_DATASOURCE_URL                          = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.sr_db_jdbc.id})"
-    APPLICATIONINSIGHTS_CONNECTION_STRING          = "InstrumentationKey=${data.azurerm_application_insights.app_insights.instrumentation_key};IngestionEndpoint=https://eastus-1.in.applicationinsights.azure.com/"
-    DATAHUB_API_KEY                                = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.datahub_api_key.id})"
-    SECRET_SLACK_NOTIFY_WEBHOOK_URL                = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.slack_notify_webhook_url.id})"
-    TWILIO_ACCOUNT_SID                             = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.twilio_account_sid.id})"
-    TWILIO_AUTH_TOKEN                              = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.twilio_auth_token.id})"
-    SIMPLE_REPORT_SENDGRID_API_KEY                 = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.sendgrid_api_key.id})"
-    SMARTY_AUTH_ID                                 = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.smarty_auth_id.id})"
-    SMARTY_AUTH_TOKEN                              = "@Microsoft.KeyVault(SecretUri=${data.azurerm_key_vault_secret.smarty_auth_token.id})"
+    SPRING_PROFILES_ACTIVE                = "azure-demo"
```

Comment:
The change to the active profiles here were intentional for demo?

Reply:
Yes, the profiles other than the main profile for the environment should all be (and are all) set in Spring Boot configuration. This actually changes nothing, it just eliminates duplication.